setup.py

```diff
@@ -0,0 +1,20 @@
+import setuptools
+
+install_requires = list(open('requirements.txt').read().splitlines())
```

Comment:
If only the `nce` package will be delivered, we only need to add *pytorch* (`torch >=0.4.0`) as `install_requires`.

The `dill` and `tqdm` are for the language model example.